### ISSUE TITLE
data dumper

### ISSUE THREAD

**Issue by Burnedfart** [MrElectrify/AssettoCorsaTools#1]
how do i use the car data dumper?

**Comment by BautistaBordes** [comment]
> how do i use the car data dumper?

I think you need to compile CarDataDumper.cpp first and then with the .exe you send the arguments, but I am getting a lot of problems with the #include in a lot of files and can't compile, so idk if that's how it works

**Comment by Burnedfart** [comment]
> > how do i use the car data dumper?
> 
> I think you need to compile CarDataDumper.cpp first and then with the .exe you send the arguments, but I am getting a lot of problems with the #include in a lot of files and can't compile, so idk if that's how it works

yeah i compiled the exe successfully but when i run it, a console appears for a frame and then disappears, i added a std::cin.get(); to make it stay on my screen and it says something about file not found, so idk how to add the file or fill in the arguments

**Comment by BautistaBordes** [comment]
> > > how do i use the car data dumper?
> > 
> > 
> > I think you need to compile CarDataDumper.cpp first and then with the .exe you send the arguments, but I am getting a lot of problems with the #include in a lot of files and can't compile, so idk if that's how it works
> 
> yeah i compiled the exe successfully but when i run it, a console appears for a frame and then disappears, i added a std::cin.get(); to make it stay on my screen and it says something about file not found, so idk how to add the file or fill in the arguments

To send the arguments open cmd and go to the same folder ( or paste the path of the .exe) where the .exe is and then write the name of the .exe and the path of the data.acd, like this: "AssettoCorsaCarDataDumper.exe C:\steam\assettocorsa\content\cars\lotus\data.acd", it should work like that
